packages/cms/store/auth.ts

```diff
@@ -46,7 +55,10 @@ export const useAuthStore = create<AuthState & AuthMethods>(
       return window.Tensei.api.post('/auth/register', credentials)
     },
     async logout() {
-      return window.Tensei.api.post('/auth/logout')
+      return window.Tensei.api.post('/logout')
+    },
+    async updateProfile(credential: UpdateProfileCredentials) {
```

Comment:
Would be `input: UpdateUserProfileInput`

Reply:
okay